### PR TITLE
Add node names to database.

### PR DESCRIPTION
**Pull request by litghost**

### CODE REVIEW

**Comment by litghost** [comment]
@andrewb1999 / @daveshah1 This will add node names to database once merged.

**Review by mkurc-ant — APPROVED** [review]
LGTM

File: fuzzers/074-dump_all/node_names.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
```
Comment (acomodi) [inline fuzzers/074-dump_all/node_names.py:1]: Missing file docstrings

File: fuzzers/074-dump_all/check_nodes.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
```
Comment (acomodi) [inline fuzzers/074-dump_all/check_nodes.py:1]: Missing file docstrings

**Review by acomodi — APPROVED** [review]
LGTM, probably worth having some docstrings in the newly added scripts.

File: fuzzers/074-dump_all/node_names.py (hunk 1 above)
Comment (litghost) [reply fuzzers/074-dump_all/node_names.py:1]: Fixed

File: fuzzers/074-dump_all/check_nodes.py (hunk 2 above)
Comment (litghost) [reply fuzzers/074-dump_all/check_nodes.py:1]: Fixed

**Pull request merged**